tests/ci/cherry_pick.py

```diff
@@ -67,15 +67,17 @@ class ReleaseBranch:
 ### Note
 
 This pull-request will be merged automatically as it reaches the mergeable state, \
-**do not merge it manually**.
+**do not merge it manually**. It's 100% safe, but completely meaningless.
 
 ### If the PR was closed and then reopened
 
-If it stuck, check {pr_url} for `{backport_created_label}` and delete it if \
-necessary. Manually merging will do nothing, since `{backport_created_label}` \
-prevents the original PR {pr_url} from being processed.
+If it stuck (e.g. for a day), check {pr_url} for `{backport_created_label}` *label* and \
+delete it if necessary. Manually merging will do nothing, since \
+`{backport_created_label}` *label* prevents the original PR {pr_url} from being \
+processed.
 
-If you want to recreate the PR: delete the `{label_cherrypick}` label and delete this branch.
+If the cherry-pick PR is completely screwed, and you want to recreate it: delete the \
```

Comment:
I guess a "competely screwed" cherry-pick PR is one with tons of merge conflicts, right? This text recommends to delete the label and delete branch.

This seems a weird recommendation. Why does it not recommend to resolve the merge conflicts and push a new version to the cherry-pick branch?